ceph.spec.in

```diff
@@ -1579,6 +1592,19 @@ if [ $1 -eq 1 ] ; then
     /usr/bin/systemctl try-restart ceph-mgr.target >/dev/null 2>&1 || :
 fi
 
+%files mgr-ssh
+%{_libdir}/ceph/mgr/ssh
```

Comment:
s/libdir/datadir/ this was changed in 5aab0999

Reply:
fixed